os/kernel/init/os_bringup.c

```diff
@@ -276,7 +276,13 @@ static inline void os_do_appstart(void)
 #ifdef CONFIG_SECURITY_LEVEL
 	int ret = set_security_level();
 	if (ret != SECURITY_LEVEL_OK) {
-		dbg("[set_security_level] error ret : %d\n", ret);
+		if (ret == SECURITY_LEVEL_READ_FAILED) {
```

Comment:
1. You print the success case in the function. But you add failure cases at out of the function. Is there any reason? If not, let's get the same policy for them.
2. Log messages occupy some flash memory. To optimize memory usage, let's remove duplicated words.

Reply:
1. fixed to pring log inside of `set_security_level`
2. I erased duplicated words.